app.js

```diff
@@ -1,117 +1,186 @@
 const express = require("express");
+const db = require("./databaseConnect");
 const app = express();
 const port = 3001;
+const jwt = require("jsonwebtoken");
+const authMiddleware = require("./authMiddleware");
+
+db.init();
 
 var hash = require("pbkdf2-password")();
 
 var cors = require("cors");
+const { json } = require("express");
 
 app.use(cors());
 
 app.use(express.urlencoded({ extended: true }));
 // This is required to handle urlencoded data
 app.use(express.json());
-// This to handle json data coming from requests mainly post
 
-var users = {
-    admin: { name: "admin" },
-};
+app.use(express.static("public"));
```

Comment:
explain what this line does, what arguments you are passing and why

Reply:
The app.use(express.static("public")); serves static files (images) from the "public" folder.